main/res/values/changelog_release.xml

```diff
@@ -2,7 +2,7 @@
 <resources>
   <!-- changelog for the release branch -->
   <string name="changelog_release" translatable="false">\n
-    <b>Next beta version:</b>\n
+    <b>2017.11.01-RC:</b>\n
```

Comment:
Looks like a merge error on your side.

Reply:
No worries.

I actually see that while I was merging and thought "2017.11.01-RC" was the correct choice.  I modify it.

Reply:
Hang-on, Lars made that change about an hour before I committed.

What should it be?

Reply:
I fixed it by changing the target branch to `release`, where it should be.

Reply:
Just so I understand, how does one set the "Target Branch"?
I take it it's not as simple as just rebasing onto the "release" branch.

Reply:
If you edit the PR on the Website. On the top you have a dropdown with the branch.

Reply:
Oh, I see.

So even though I'd rebased onto 'release' the actual request was still being made to 'master'.  So I need to both 'rebase' _then_ update the target branch.  Got it 👍